docs/source/explanation/Main_algorithms/multi_label_classification.rst

```diff
@@ -0,0 +1,89 @@
+
+We also support our custom and simple dataset format for multi-label classificaiton. It has the following structure:
+
+::
+
+    data
+    ├── images
+        ├── train
+            ├── 0.png
+            ├── 1.png
+            ...
+            └── N.png
+        ├── val
+            ├── 0.png
+            ├── 1.png
+            ...
+            └── N.png
+    └── annotations
+        ├── train.json
+        └── val.json
+
+Where ``*.json`` annotations have two keys: **"images"** and **"classes"**. Key **"images"** includes lists of unique images with lists of classes presented on this image. Key **"classes"** consists of list with all classes in the dataset.
+Below a simple example of 2 classes presented:
+
+::
+
+    {"images": [
+        ["0.png", ["class_0"]],
+        ["Slide19.PNG", ["class_0"]],
+        ["Slide16.PNG", ["class_0", "class_1"]]
+    ],
+    "classes": ["class_0", "class_1"]}
+
+
+*********
+Backbones
+*********
+We use the same model templates as for multi-class classification. Please, refer: :doc:`multi_class_classification`
+
+*********
+Supervised Incremental Learning
+*********
+
+The main goal of the task is to predict a set of labels per image. We solve this problem by optimizing small binary classification sub-tasks whether or not the exact class is presented on the given image.
+
+For supervised learning we use the following algorithms components:
+
+- ``Augmentations``: Besides basic augmentations like random flip and random rotate, we use `Augmix <https://arxiv.org/abs/1912.02781>`_. This advanced type of augmentation helps to significantly expand the training distribution.
+
+- ``Optimizer``: `Sharpness Aware Minimization (SAM) <https://arxiv.org/abs/2209.06585>`_. Wrapper upon the SGD that helps to achieve better generalization minimizing simultaneously loss value and loss sharpness.
+
+- ``Learning rate schedule``: `One Cycle Learning Rate policy <https://arxiv.org/abs/1708.07120>`_. It is the combination of gradually increasing the learning rate and gradually decreasing the momentum during the first half of the cycle, then gradually decreasing the learning rate and increasing the momentum during the latter half of the cycle.
+
+- ``Loss function``: We use **Asymmetric Angular Margin Loss**. We can formulate this loss as follows: :math:`L_j (cos\Theta_j,y) = \frac{k}{s}y p_-^{\gamma^-}\log{p_+} + \frac{1-k}{s}(1-y)p_+^{\gamma^+}\log{p_-}`, where :math:`s` is a scale parameter, :math:`m` is an angular margin, :math:`k` is negative-positive weighting coefficient, :math:`\gamma^+` and :math:`\gamma^-` are weighting parameters. For further information about loss function, ablation studies, and experiments, please refer to our dedicated `paper <https://arxiv.org/abs/2209.06585>`_.
+
+Additionally, we use `No Bias Decay (NBD) <https://arxiv.org/abs/1812.01187>`_ technique, **Exponential Moving Average (EMA)** for the model`s weights and adaptive **early stopping** to add adaptability and prevent overfitting.
```

Comment:
Link for `Exponential Moving Average (EMA)`?

Reply:
Unfortunately, we don't have this one, I guess.